mmv1/third_party/terraform/services/redis/resource_redis_cluster_test.go

```diff
@@ -1418,3 +1418,103 @@ resource "google_redis_cluster" "cluster-ms" {
 }
 `, context)
 }
+
+func TestAccRedisCluster_redisClusterHaWithLabelsUpdate(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
```

Comment:
```suggestion
		"deletion_protection_enabled": false,
		"random_suffix": acctest.RandString(t, 10),
```

Reply:
I'm gonna go ahead and apply this since I think it's the last change and pretty trivial